tests/candidateApp-advanced.spec.ts

```diff
@@ -1,33 +1,59 @@
 import { test, expect } from "@playwright/test";
 import { faker } from "@faker-js/faker";
 import path from "path";
+import { TRANSLATIONS as T } from './utils/translations';
```

Comment:
Use just a single import that contains all the translations instead of importing individual json files.